docs/main/user-guides/replicated-storage-user-guide/advanced-operations/monitoring.md

```diff
@@ -101,4 +101,107 @@ Upon successful integration of the exporter with the Prometheus stack, the metri
 | kubelet_volume_stats_inodes_used | Gauge | Integer | The total number of inodes that have been utilized to store metadata. |
 
 
+## Performance Monitoring Stack
+
+Earlier, the pool capacity/state stats were exported, and the exporter used to cache the metrics and return when Prometheus client queried. This was not ensuring the latest data retuns during the Prometheus poll cycle.
+
+This feature exports performace stats for Volume, Replica and DiskPool resource. Mayastor Prometheus exporter runs along with IO engine container and exposes Prometheus counters. In addition to that, exporter does not pre-populate the metrics. It queries the IO engine, updates cache, and returns the latest metrics when the Prometheus client queries the exporter.
+
+With this change, the exporter calls to the IO engine for stats tied to the frequency of Prometheus poll cycle.
```

Comment:
Can this sentence be grammatically improved?

Reply:
Is the below sentence fine @avishnu 

"With this change, the exporter will query the IO engine for stats related to the frequency of Prometheus poll cycle."

Reply:
'the stats related to the frequency..' is not correct.

Reply:
Is it "stats tied to the frequency"? @avishnu